src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1303,6 +1303,40 @@ private bool CanAddToGroup(object parameters)
             return false;
         }
 
+        private void SelectUpstream(object parameters)
+        {
+            NodeModel.SelectUpStreamNeighbours();
+        }
+
+        private bool CanSelectUpstream(object parameters)
+        {
+            //Add checks for can stream
+            return true;
+        }
+
+        private void SelectDownstream(object parameters)
+        {
+            NodeModel.SelectDownStreamNeighbours();
+        }
+
+        private bool CanSelectDownstream(object parameters)
+        {
+            //Add checks for can stream
+            return true;
+        }
+
+        private void SelectDownAndUpstream(object parameters)
```

Comment:
Should we add neighbours to this, otherwise its a bit hard to know what we are actually selecting based on the name